src/partials/home/get-started.js

```diff
@@ -33,16 +33,16 @@ const StyledLandingSectionContent = styled(LandingSectionContent)`
 
 const GetStarted = ({ description, link }) => {
   return (
-    <LandingSectionWrapper bg={importedTheme.color.red}>
+    <LandingSectionWrapper bg={importedTheme.color.tabasco}>
       <StyledLandingSectionContent color={importedTheme.color.white}>
         <SectionHeading>Get Started</SectionHeading>
         <SectionDescription>{description}</SectionDescription>
-        <LinkButton to={link.location} color={importedTheme.color.darkBrown}>
+        <LinkButton to={link.location} color={importedTheme.color.cabSav}>
           {link.text}
         </LinkButton>
       </StyledLandingSectionContent>
-      <Stripe bg={importedTheme.color.brown} />
-      <Stripe bg={importedTheme.color.otherBrown} />
+      <Stripe bg={importedTheme.color.rouge} />
+      <Stripe bg={importedTheme.color.crownOfThorns} />
```

Comment:
😂

Reply:
I know right?!? I swear I didn't make it up.
<img width="1017" alt="Screen Shot 2020-02-21 at 5 54 08 PM" src="https://user-images.githubusercontent.com/10720454/75082909-2db6aa80-54d3-11ea-8ca5-52e5f62722cf.png">